pkg/drivers/powermax.go

```diff
@@ -0,0 +1,258 @@
+package drivers
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	acorev1 "k8s.io/client-go/applyconfigurations/core/v1"
+
+	csmv1 "github.com/dell/csm-operator/api/v1"
+	"github.com/dell/csm-operator/pkg/logger"
+	"github.com/dell/csm-operator/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Constant to be used for powermax deployment
+const (
+	// PowerMaxPluginIdentifier used to identify powermax plugin
+	PowerMaxPluginIdentifier = "powermax"
```

Comment:
Do we need this here or can we use the same one defined in types.go

Reply:
It is needed here as all the modules point to <driver>PluginIdentifier for its value.